GameData/KerbalismConfig/System/Parts.cfg

```diff
@@ -976,41 +977,72 @@
 
     SETUP
     {
-      name = Fuel Cell
-      desc = A fuel cell that combines <b>Liquid Hydrogen</b> and <b>Liquid Oxygen</b>, producing <b>Water</b> as a by-product.
-      tech = maturePower
+      name = Air Pump
+      desc = Pumps external atmosphere into the cabin to maintan internal pressure. Only works in breathable atmospheres
+      tech = crewSurvivability
 
       MODULE
       {
         type = ProcessController
         id_field = resource
-        id_value = _FuelCell
+        id_value = _AirPump
       }
     }
     SETUP
     {
-      name = Air Pump
-      desc = Pumps external atmosphere into the cabin to maintan internal pressure. Only works in breathable atmospheres
-      tech = crewSurvivability
+      name = N2 Pressure Control
+      desc = Use <b>Nitrogen</b> to maintain the internal atmosphere at a comfortable pressure.
+      tech = earlyLifeSupport
 
       MODULE
       {
         type = ProcessController
         id_field = resource
-        id_value = _AirPump
+        id_value = _PressureControl
       }
     }
     SETUP
     {
-      name = N2 Pressure Control
-      desc = Use <b>Nitrogen</b> to maintain the internal atmosphere at a comfortable pressure.
+      name = LOX to GOX Converter
+      desc = Heats <b>LqdOxygen</b> to breathable <b>Oxygen</b>.
       tech = earlyLifeSupport
 
       MODULE
       {
         type = ProcessController
         id_field = resource
-        id_value = _PressureControl
+        id_value = _LOXConverter
+      }
+    }
+  }
+}
+
+//Add fuel cells to Mk3 here instead
+@PART[mk3Cockpit_Shuttle]:NEEDS[ProfileRealismOverhaul]:AFTER[RealismOverhaul]
+{
+  MODULE
+  {
+    name = ProcessController
+    resource = _FuelCell
+    title = Fuel Cell
+    capacity = 7
```

Comment:
Shouldn't all shuttles get 21 kW, if you found a source stating 21 kW?

Reply:
There are 3 fuel cells, total 21 kW

Reply:
but isn't this adding just one?